### PR TITLE
[FSPE-6495] Added tooltips to header component documentation examples

### PR DESCRIPTION
**Pull request by shintuviswambharan**
Added tooltip to chi header component example 
@janibolkvadze  could you please review

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://ecdd0645d9895abc13e3c59e2fd7d142eda6a0bd-chi-pr.enterprise.ctl.io" target="_blank"> https://ecdd0645d9895abc13e3c59e2fd7d142eda6a0bd-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1561/ for more details.

File: src/website/views/components/header/_examples.pug [hunk 1]
```diff
@@ -500,11 +504,11 @@ h3 Portal Header with Impersonation Bar
                 span.chi-dropdown__menu-item_text App description
         .chi-header__end
           div.-d--none.-d-lg--flex
-            button.chi-button.-flat.-icon(aria-label="Notifcations")
+            button.chi-button.-flat.-icon(id="button-ImpBar-notification" aria-label="Notifcations" data-tooltip="Notifications")
```
Comment (janibolkvadze) [inline src/website/views/components/header/_examples.pug:507]:
```suggestion
            button.chi-button.-flat.-icon(id="button-impersonation-notification" aria-label="Notifcations" data-tooltip="Notifications")
```

File: src/website/views/components/header/_examples.pug [hunk 2]
```diff
@@ -500,11 +504,11 @@ h3 Portal Header with Impersonation Bar
...
               .chi-button__content
                 i.chi-icon.icon-bell-outline(aria-hidden="true")
                 div.chi-badge.-primary.-xs 2
-            button.chi-button.-flat.-icon(aria-label="Support")
+            button.chi-button.-flat.-icon(id="button-ImpBar-support" aria-label="Support" data-tooltip="Support")
```
Comment (janibolkvadze) [inline src/website/views/components/header/_examples.pug:511]:
```suggestion
            button.chi-button.-flat.-icon(id="button-impersonation-support" aria-label="Support" data-tooltip="Support")
```

File: src/website/views/components/header/_examples.pug [hunk 3]
```diff
@@ -587,13 +591,13 @@ h3 Portal Header with Impersonation Bar
           </div>
           <div class="chi-header__end">
             <div class="-d--none -d-lg--flex">
-              <button class="chi-button -flat -icon" aria-label="Notifcations">
+              <button class="chi-button -flat -icon" id="button-ImpBar-notification" aria-label="Notifcations" data-tooltip="Notifications">
```
Comment (janibolkvadze) [inline src/website/views/components/header/_examples.pug:594]:
```suggestion
              <button class="chi-button -flat -icon" id="button-impersonation-notification" aria-label="Notifcations" data-tooltip="Notifications">
```

File: src/website/views/components/header/_examples.pug [hunk 4]
```diff
@@ -587,13 +591,13 @@ h3 Portal Header with Impersonation Bar
...
                 <div class="chi-button__content">
                   <i class="chi-icon icon-bell-outline" aria-hidden="true"></i>
                   <div class="chi-badge -primary -xs">2</div>
                 </div>
               </button>
-              <button class="chi-button -flat -icon" aria-label="Support">
+              <button class="chi-button -flat -icon" id="button-ImpBar-support" aria-label="Support" data-tooltip="Notifications">
```
Comment (janibolkvadze) [inline src/website/views/components/header/_examples.pug:600]:
```suggestion
              <button class="chi-button -flat -icon" id="button-impersonation-support" aria-label="Support" data-tooltip="Notifications">
```

File: src/website/views/components/header/_examples.pug [hunk 5]
```diff
@@ -655,6 +659,8 @@ h3 Portal Header with Impersonation Bar
         chi.dropdown(document.getElementById('button-user-menu'));
         chi.dropdown(document.getElementById('button-eid-menu'));
         chi.tooltip(document.getElementById('button-logout'));
+        chi.tooltip(document.getElementById('button-ImpBar-notification'));
+        chi.tooltip(document.getElementById('button-ImpBar-support'));
```
Comment (janibolkvadze) [inline src/website/views/components/header/_examples.pug:663]:
```suggestion
        chi.tooltip(document.getElementById('button-impersonation-notification'));
        chi.tooltip(document.getElementById('button-impersonation-support'));
```

File: src/website/views/components/header/_examples.pug [hunk 6]
```diff
@@ -1258,6 +1264,12 @@ script.
       chi.dropdown(document.getElementById('button-eid-menu2'));
       chi.dropdown(document.getElementById('button-eid-menu3'));
       chi.tooltip(document.getElementById('button-logout'));
+      chi.tooltip(document.getElementById('button-notification'));
+      chi.tooltip(document.getElementById('button-support'));
+      chi.tooltip(document.getElementById('button-nav-notification'));
+      chi.tooltip(document.getElementById('button-nav-support'));
+      chi.tooltip(document.getElementById('button-ImpBar-notification'));
+      chi.tooltip(document.getElementById('button-ImpBar-support'));
```
Comment (janibolkvadze) [inline src/website/views/components/header/_examples.pug:1272]:
```suggestion
      chi.tooltip(document.getElementById('button-impersonation-notification'));
      chi.tooltip(document.getElementById('button-impersonation-support'));
```

**Review by janibolkvadze — CHANGES_REQUESTED** [review]

**Review by janibolkvadze — APPROVED** [review]

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://c5bc065f9bf302e0ef574e050505295251568d96-chi-pr.enterprise.ctl.io" target="_blank"> https://c5bc065f9bf302e0ef574e050505295251568d96-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1566/ for more details.

File: src/website/views/components/header/_examples.pug [hunk 7]
```diff
@@ -106,11 +106,11 @@ h3 Portal Header
                 span.chi-dropdown__menu-item_text App description
         .chi-header__end
           div.-d--none.-d-lg--flex
-            button.chi-button.-flat.-icon(aria-label="Notifcations")
+            button.chi-button.-flat.-icon(id="button-notification" aria-label="Notifcations" data-tooltip="Notifications")
```
Comment (jllr) [inline src/website/views/components/header/_examples.pug:109]:
```suggestion
            button.chi-button.-flat.-icon(id="button-notification" aria-label="Notifcations" data-tooltip="Notifications" data-position="bottom")
```

File: src/website/views/components/header/_examples.pug [hunk 8]
```diff
@@ -106,11 +106,11 @@ h3 Portal Header
...
               .chi-button__content
                 i.chi-icon.icon-bell-outline(aria-hidden="true")
                 div.chi-badge.-primary.-xs 2
-            button.chi-button.-flat.-icon(aria-label="Support")
+            button.chi-button.-flat.-icon(id="button-support" aria-label="Support" data-tooltip="Support")
```
Comment (jllr) [inline src/website/views/components/header/_examples.pug:113]:
```suggestion
            button.chi-button.-flat.-icon(id="button-support" aria-label="Support" data-tooltip="Support" data-position="bottom")
```

File: src/website/views/components/header/_examples.pug [hunk 9]
```diff
@@ -181,13 +181,13 @@ h3 Portal Header
           </div>
           <div class="chi-header__end">
             <div class="-d--none -d-lg--flex">
-              <button class="chi-button -flat -icon" aria-label="Notifcations">
+              <button class="chi-button -flat -icon" id="button-notification" aria-label="Notifcations" data-tooltip="Notifications">
```
Comment (jllr) [inline src/website/views/components/header/_examples.pug:184]:
```suggestion
              <button class="chi-button -flat -icon" id="button-notification" aria-label="Notifcations" data-tooltip="Notifications" data-position="bottom">
```

File: src/website/views/components/header/_examples.pug [hunk 10]
```diff
@@ -181,13 +181,13 @@ h3 Portal Header
...
                 <div class="chi-button__content">
                   <i class="chi-icon icon-bell-outline" aria-hidden="true"></i>
                   <div class="chi-badge -primary -xs">2</div>
                 </div>
               </button>
-              <button class="chi-button -flat -icon" aria-label="Support">
+              <button class="chi-button -flat -icon" id="button-support" aria-label="Support" data-tooltip="Support">
```
Comment (jllr) [inline src/website/views/components/header/_examples.pug:190]:
```suggestion
              <button class="chi-button -flat -icon" id="button-support" aria-label="Support" data-tooltip="Support" data-position="bottom">
```

File: src/website/views/components/header/_examples.pug [hunk 11]
```diff
@@ -278,11 +280,11 @@ h3 Portal Header with Navbar
                 span.chi-dropdown__menu-item_text App description
         .chi-header__end
           div.-d--none.-d-lg--flex
-            button.chi-button.-flat.-icon(aria-label="Notifcations")
+            button.chi-button.-flat.-icon(id="button-nav-notification" aria-label="Notifications" data-tooltip="Notifications")
```
Comment (jllr) [inline src/website/views/components/header/_examples.pug:283]:
```suggestion
            button.chi-button.-flat.-icon(id="button-nav-notification" aria-label="Notifications" data-tooltip="Notifications" data-position="bottom")
```

File: src/website/views/components/header/_examples.pug [hunk 12]
```diff
@@ -278,11 +280,11 @@ h3 Portal Header with Navbar
...
               .chi-button__content
                 i.chi-icon.icon-bell-outline(aria-hidden="true")
                 div.chi-badge.-primary.-xs 2
-            button.chi-button.-flat.-icon(aria-label="Support")
+            button.chi-button.-flat.-icon(id="button-nav-support" aria-label="Support" data-tooltip="Support")
```
Comment (jllr) [inline src/website/views/components/header/_examples.pug:287]:
```suggestion
            button.chi-button.-flat.-icon(id="button-nav-support" aria-label="Support" data-tooltip="Support" data-position="bottom")
```

File: src/website/views/components/header/_examples.pug [hunk 13]
```diff
@@ -371,13 +373,13 @@ h3 Portal Header with Navbar
           </div>
           <div class="chi-header__end">
             <div class="-d--none -d-lg--flex">
-              <button class="chi-button -flat -icon" aria-label="Notifcations">
+              <button class="chi-button -flat -icon" id="button-nav-notification" aria-label="Notifcations" data-tooltip="Notifications">
```
Comment (jllr) [inline src/website/views/components/header/_examples.pug:376]:
```suggestion
              <button class="chi-button -flat -icon" id="button-nav-notification" aria-label="Notifcations" data-tooltip="Notifications" data-position="bottom">
```

File: src/website/views/components/header/_examples.pug [hunk 14]
```diff
@@ -371,13 +373,13 @@ h3 Portal Header with Navbar
...
                 <div class="chi-button__content">
                   <i class="chi-icon icon-bell-outline" aria-hidden="true"></i>
                   <div class="chi-badge -primary -xs">2</div>
                 </div>
               </button>
-              <button class="chi-button -flat -icon" aria-label="Support">
+              <button class="chi-button -flat -icon" id="button-nav-support" aria-label="Support" data-tooltip="Support">
```
Comment (jllr) [inline src/website/views/components/header/_examples.pug:382]:
```suggestion
              <button class="chi-button -flat -icon" id="button-nav-support" aria-label="Support" data-tooltip="Support" data-position="bottom">
```

File: src/website/views/components/header/_examples.pug [hunk 15]
```diff
@@ -500,11 +504,11 @@ h3 Portal Header with Impersonation Bar
                 span.chi-dropdown__menu-item_text App description
         .chi-header__end
           div.-d--none.-d-lg--flex
-            button.chi-button.-flat.-icon(aria-label="Notifcations")
+            button.chi-button.-flat.-icon(id="button-impersonation-notification" aria-label="Notifcations" data-tooltip="Notifications")
```
Comment (jllr) [inline src/website/views/components/header/_examples.pug:507]:
```suggestion
            button.chi-button.-flat.-icon(id="button-impersonation-notification" aria-label="Notifcations" data-tooltip="Notifications" data-position="bottom")
```

File: src/website/views/components/header/_examples.pug [hunk 16]
```diff
@@ -500,11 +504,11 @@ h3 Portal Header with Impersonation Bar
...
               .chi-button__content
                 i.chi-icon.icon-bell-outline(aria-hidden="true")
                 div.chi-badge.-primary.-xs 2
-            button.chi-button.-flat.-icon(aria-label="Support")
+            button.chi-button.-flat.-icon(id="button-impersonation-support" aria-label="Support" data-tooltip="Support")
```
Comment (jllr) [inline src/website/views/components/header/_examples.pug:511]:
```suggestion
            button.chi-button.-flat.-icon(id="button-impersonation-support" aria-label="Support" data-tooltip="Support" data-position="bottom")
```

File: src/website/views/components/header/_examples.pug [hunk 17]
```diff
@@ -587,13 +591,13 @@ h3 Portal Header with Impersonation Bar
           </div>
           <div class="chi-header__end">
             <div class="-d--none -d-lg--flex">
-              <button class="chi-button -flat -icon" aria-label="Notifcations">
+              <button class="chi-button -flat -icon" id="button-impersonation-notification" aria-label="Notifcations" data-tooltip="Notifications">
```
Comment (jllr) [inline src/website/views/components/header/_examples.pug:594]:
```suggestion
              <button class="chi-button -flat -icon" id="button-impersonation-notification" aria-label="Notifcations" data-tooltip="Notifications" data-position="bottom">
```

File: src/website/views/components/header/_examples.pug [hunk 18]
```diff
@@ -587,13 +591,13 @@ h3 Portal Header with Impersonation Bar
...
                 <div class="chi-button__content">
                   <i class="chi-icon icon-bell-outline" aria-hidden="true"></i>
                   <div class="chi-badge -primary -xs">2</div>
                 </div>
               </button>
-              <button class="chi-button -flat -icon" aria-label="Support">
+              <button class="chi-button -flat -icon" id="button-impersonation-support" aria-label="Support" data-tooltip="Notifications">
```
Comment (jllr) [inline src/website/views/components/header/_examples.pug:600]:
```suggestion
              <button class="chi-button -flat -icon" id="button-impersonation-support" aria-label="Support" data-tooltip="Notifications" data-position="bottom">
```

**Review by jllr — CHANGES_REQUESTED** [review]

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://eb9e152f796c52deb421230645724f322becc0d9-chi-pr.enterprise.ctl.io" target="_blank"> https://eb9e152f796c52deb421230645724f322becc0d9-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1571/ for more details.

File: src/website/views/components/header/_examples.pug [hunk 19]
```diff
@@ -106,11 +106,11 @@ h3 Portal Header
                 span.chi-dropdown__menu-item_text App description
         .chi-header__end
           div.-d--none.-d-lg--flex
-            button.chi-button.-flat.-icon(aria-label="Notifcations")
+            button.chi-button.-flat.-icon(id="button-notification" aria-label="Notifcations" data-tooltip="Notifications" data-position="bottom")
```
Comment (jllr) [inline src/website/views/components/header/_examples.pug:109]:
```suggestion
            button.chi-button.-flat.-icon(id="button-notification" aria-label="Notifications" data-tooltip="Notifications" data-position="bottom")
```

File: src/website/views/components/header/_examples.pug [hunk 20]
```diff
@@ -181,13 +181,13 @@ h3 Portal Header
           </div>
           <div class="chi-header__end">
             <div class="-d--none -d-lg--flex">
-              <button class="chi-button -flat -icon" aria-label="Notifcations">
+              <button class="chi-button -flat -icon" id="button-notification" aria-label="Notifcations" data-tooltip="Notifications" data-position="bottom">
```
Comment (jllr) [inline src/website/views/components/header/_examples.pug:184]:
```suggestion
              <button class="chi-button -flat -icon" id="button-notification" aria-label="Notifications" data-tooltip="Notifications" data-position="bottom">
```

File: src/website/views/components/header/_examples.pug [hunk 21]
```diff
@@ -371,13 +373,13 @@ h3 Portal Header with Navbar
           </div>
           <div class="chi-header__end">
             <div class="-d--none -d-lg--flex">
-              <button class="chi-button -flat -icon" aria-label="Notifcations">
+              <button class="chi-button -flat -icon" id="button-nav-notification" aria-label="Notifcations" data-tooltip="Notifications" data-position="bottom">
```
Comment (jllr) [inline src/website/views/components/header/_examples.pug:376]:
```suggestion
              <button class="chi-button -flat -icon" id="button-nav-notification" aria-label="Notifications" data-tooltip="Notifications" data-position="bottom">
```

File: src/website/views/components/header/_examples.pug [hunk 22]
```diff
@@ -500,11 +504,11 @@ h3 Portal Header with Impersonation Bar
                 span.chi-dropdown__menu-item_text App description
         .chi-header__end
           div.-d--none.-d-lg--flex
-            button.chi-button.-flat.-icon(aria-label="Notifcations")
+            button.chi-button.-flat.-icon(id="button-impersonation-notification" aria-label="Notifcations" data-tooltip="Notifications" data-position="bottom")
```
Comment (jllr) [inline src/website/views/components/header/_examples.pug:507]:
```suggestion
            button.chi-button.-flat.-icon(id="button-impersonation-notification" aria-label="Notifications" data-tooltip="Notifications" data-position="bottom")
```

File: src/website/views/components/header/_examples.pug [hunk 23]
```diff
@@ -587,13 +591,13 @@ h3 Portal Header with Impersonation Bar
           </div>
           <div class="chi-header__end">
             <div class="-d--none -d-lg--flex">
-              <button class="chi-button -flat -icon" aria-label="Notifcations">
+              <button class="chi-button -flat -icon" id="button-impersonation-notification" aria-label="Notifcations" data-tooltip="Notifications" data-position="bottom">
```
Comment (jllr) [inline src/website/views/components/header/_examples.pug:594]:
```suggestion
              <button class="chi-button -flat -icon" id="button-impersonation-notification" aria-label="Notifications" data-tooltip="Notifications" data-position="bottom">
```

File: src/website/views/components/header/_examples.pug [hunk 24]
```diff
@@ -587,13 +591,13 @@ h3 Portal Header with Impersonation Bar
...
                 <div class="chi-button__content">
                   <i class="chi-icon icon-bell-outline" aria-hidden="true"></i>
                   <div class="chi-badge -primary -xs">2</div>
                 </div>
               </button>
-              <button class="chi-button -flat -icon" aria-label="Support">
+              <button class="chi-button -flat -icon" id="button-impersonation-support" aria-label="Support" data-tooltip="Notifications" data-position="bottom">
```
Comment (jllr) [inline src/website/views/components/header/_examples.pug:600]:
```suggestion
              <button class="chi-button -flat -icon" id="button-impersonation-support" aria-label="Support" data-tooltip="Support" data-position="bottom">
```

**Review by jllr — CHANGES_REQUESTED** [review]

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://d594e6b62b0805d54f68e875c95795c01253249a-chi-pr.enterprise.ctl.io" target="_blank"> https://d594e6b62b0805d54f68e875c95795c01253249a-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1575/ for more details.

**Review by jllr — APPROVED** [review]

**Comment by allyourbotsarebelongtous** [comment]
The following instances are being terminated: https://ops.elasticbox.com/services/instances/i-3ll78k, https://ops.elasticbox.com/services/instances/i-e3gtoz, https://ops.elasticbox.com/services/instances/i-t7hwml, https://ops.elasticbox.com/services/instances/i-qlmvdv

**Pull request merged**